### PR TITLE
Cc 0009 sign in by using email and password

### PR DESCRIPTION
**Pull request by VanderDeng**
Project initialization
Including:
[chore: add dependencies]
[docs: add login related images]
[docs: delete useless style files]
[docs: README update]
[docs: delete useless layout&components]
[feat(api): init api & environment]
[feat(home page): add homepage & update url]
[feat(Login): cc-0009 login created]
[refactor: abstract action & deconstruct data]
[Merge branch 'main' into cc-0009-Sign-in-by-using-Email-and-Password]
[Merge branch 'main' into cc-0009-Sign-in-by-using-Email-and-Password]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
 This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
+### Build for Deploy
+
+```shell
+$ npm run build
+# or
+$ yarn build
+```
```
Comment (tatoMa) [inline README.md:9]: We might need a section of export static files for deployment as we are currently using S3 to store this web app.

Comment (tatoMa) [reply README.md:9]: Have a look at cc-app repo

File: package.json [hunk 2]
```diff
@@ -20,14 +20,18 @@
     "prepare": "husky install"
   },
   "dependencies": {
+    "@chakra-ui/icons": "^1.1.7",
```
Comment (tatoMa) [inline package.json:23]: Hoping you guys stick to only one icon lib.

Comment (VanderDeng) [reply package.json:23]: Can we subsequently add more icon libraries as needed?

File: README.md (hunk 1 above)
Comment (VanderDeng) [reply README.md:9]: got it, I will add this one

**Pull request closed without merging**